frontend/src/views/c-authorship.vue

```diff
@@ -449,6 +465,10 @@ export default {
       this.setInfoHash();
     },
 
+    getAuthors(file) {
+      return [...new Set(file.segments.map((segment) => segment.knownAuthor).filter(Boolean))];
```

Comment:
Just to check, what does the spread `...` operator do here in this case?

Reply:
This notation helps convert the contents of the set to an array. 
But since `v-for` works for Sets as well in Vue 3, I will remove this conversion to improve performance.